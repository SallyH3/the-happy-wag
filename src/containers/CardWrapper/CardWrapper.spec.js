import React from 'react';
import { CardWrapper } from './index';
import { shallow } from 'enzyme';

describe('CardWrapper', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <CardWrapper />
    )
  })

  it.skip('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})