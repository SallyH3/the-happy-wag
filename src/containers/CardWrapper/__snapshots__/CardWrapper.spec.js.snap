// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardWrapper should match snapshot 1`] = `
<section
  className="card-container"
>
  <Card
    age="Baby"
    contact={
      Object {
        "email": "furrfoundation@gmail.com",
      }
    }
    gender="Male"
    id={44853380}
    key="44853380"
    name="Marcel"
    photo={
      Array [
        Object {},
      ]
    }
    species="Cat"
  />
</section>
`;
